lib/assessment.rb

```diff
@@ -13,28 +19,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce{ |memo, item| memo + item }
```

Comment:
Yep. Could also have been written 
`enum.reduce(&:+)` (or, as of Ruby 2.0, `enum.reduce(:+)`)